datacenter/dtr/2.2/guides/admin/upgrade.md

```diff
@@ -45,6 +45,12 @@ Before starting your upgrade planning, make sure that the version of UCP you are
 using is supported by the version of DTR you are trying to upgrade to. <!--(TODO:
 link to the compatibility matrix)-->
 
+> **Warning**
+>
+> Before performing any upgrade it’s important to backup. See 
+> [DTR backups and recovery](/datacenter/dtr/2.2/guides/admin/backups-and-disaster-recovery/.md).
```

Comment:
Can you link to `/datacenter/dtr/2.2/guides/admin/backups-and-disaster-recovery/`
 instead? It provides a bit more guidance than just the reference.